Doc/library/io.rst

```diff
@@ -77,7 +77,7 @@ objects.  No encoding, decoding, or newline translation is performed.  This
 category of streams can be used for all kinds of non-text data, and also when
 manual control over the handling of text data is desired.
 
-The easiest way to create a binary stream is with :meth:`open()` with ``'b'`` in
+The easiest way to create a binary stream is with :meth:`~io.open` with ``'b'`` in
```

Comment:
```suggestion
The easiest way to create a binary stream is with :meth:`open()` with ``'b'`` in
```

Reply:
I think the :meth: role automatically adds the parentheses afterwards 🙂